pip_audit/service/__init__.py

```diff
@@ -0,0 +1,5 @@
+from pip_audit.service.service import (  # noqa: F401
```

Comment:
There's something weird going on with this file. If I do a `make lint`, it always ends up reformatting this file. I think something in the lint chain is modifying this file in a way that an earlier step doesn't agree with.

Reply:
Yeah, this is probably my fault -- I need to tweak the `isort` configs to compensate for `black`'s preferences. I'll jump into your PR to do that.

Reply:
Hmm, I wasn't able to repro -- `make lint` doesn't change anything for me unless I remove the `noqa`, which we should keep since we're doing a re-export here. Maybe a version mismatch?